validation-phase-workflow.cwl

```diff
@@ -110,49 +150,37 @@ steps:
     out: [finished]
 
   check_status:
-    run: https://raw.githubusercontent.com/Sage-Bionetworks/ChallengeWorkflowTemplates/v3.1/cwl/check_status.cwl
+    run: https://raw.githubusercontent.com/Sage-Bionetworks/ChallengeWorkflowTemplates/v4.1/cwl/check_status.cwl
     in:
       - id: status
         source: "#validate/status"
       - id: previous_annotation_finished
         source: "#annotate_validation_with_output/finished"
       - id: previous_email_finished
-        source: "#email_validation/finished"
+        source: "#validation_email/finished"
     out: [finished]
-
-  score:
-    run: steps/score.cwl
+ 
+  archive:
+    run: archive.cwl
```

Comment:
```suggestion
    run: writeup/archive.cwl
```

Unless there's supposed to be another `archive.cwl` file in the same directory as this workflow.